spec/features/contextual_navigation_spec.rb

```diff
@@ -145,15 +145,9 @@ def then_i_see_the_taxonomy_sidebar_and_collection
     # The taxonomy sidebar is currently render via a shared slimmer component.
```

Comment:
We can remove these comments now the sidebar has moved and the tests have been updated?